export-airtable-rules.ts

```diff
@@ -0,0 +1,84 @@
+import * as fs from 'fs'
+import * as yaml from 'js-yaml'
+import Airtable from 'airtable'
+
+require('dotenv').config()
+
+type Dict = {
+  [key: string]: string | number | string[] | Spec[]
```

Comment:
```suggestion
  [key: string]: string | string[] | Spec[]
```

numberは想定していない作りっぽいのでけずってしまってよいかも！

Reply:
同じくこのコミットで適用されていることを確認しました！！
Fixed in 687507f45cbed882eb3087d5e46c01c76af7d2fe